content.md

```diff
@@ -69,6 +69,9 @@ A collection of tools and resources to enhance your gameplay!
 - [osu! Web Beatmap Viewer](https://preview.tryz.id.vn/) - View a beatmap in an editor-like interface in the browser
 - [Agka's SV Tools](https://zardoru.github.io/sv-tools/) - Tools to help create and edit Scroll Velocities for the osu!mania mode
 ---
+### Profile Tools
```

Comment:
Unless theres other items that could go here, its better if it stays within misc, having a category dedicated to one item is excessive.

Reply:
Makes sense!

I moved it to misc 👍